packages/kit/src/core/build/index.js

```diff
@@ -488,12 +488,10 @@ async function build_server(
 			}
 		},
 		ssr: {
-			noExternal: [
-				'@sveltejs/kit', // TODO: see https://github.com/vitejs/vite/issues/3953
```

Comment:
we still need it in build, right? i'd think build and dev should be the same

Reply:
The issue says it works on build using `adapter-node`, not tested with other adapters yet, but I guess it makes sense to have them on both.